src/components/funFacts/FunFactPage.scss

```diff
@@ -0,0 +1,51 @@
+@import "../../constants.scss";
+
+h1 {
+    padding-left: 80px;
+    margin-top: 14px;
+}
+
+.funfact {
```

Comment:
It might help you to know you can hit ctrl+shift+H to bring up a dialog that allows you to find and replace strings over multiple files, to replace all your occurrences of `funfact` with `fun-fact`

Reply:
done, thanks